egs2/vctk/asr1/local/data_download.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
```

Comment:
If it is identical to vctk/tts1, how about use soft link instead of copying that. So that we can easily sync the updates, once there is an update. (same for data_prep)

Reply:
done